Lib/calendar.py

```diff
@@ -726,6 +724,9 @@ def main(args):
     locale = options.locale, options.encoding
 
     if options.type == "html":
+        if options.month:
+            parser.error("incorrect number of arguments")
+            sys.exit(1)
```

Comment:
And on line 722
```suggestion
            raise SystemExit(1)
```

Reply:
This is outside of the scope of this PR.
BTW. May I ask why you want this change?

Reply:
Second count of my error, I had thought you'd added the `sys.exit` call. As to why, it saves indirection and a function call -- `sys.exit` uses `SystemExit` internally. You can see that it takes fewer instructions in `dis` output:

```pycon
>>> def f(): sys.exit(1)
>>> def g(): raise SystemExit(1)
>>> dis.dis(f)
  1           0 LOAD_GLOBAL              0 (sys)
              2 LOAD_METHOD              1 (exit)
              4 LOAD_CONST               1 (1)
              6 CALL_METHOD              1
              8 POP_TOP
             10 LOAD_CONST               0 (None)
             12 RETURN_VALUE
>>> dis.dis(g)
  1           0 LOAD_GLOBAL              0 (SystemExit)
              2 LOAD_CONST               1 (1)
              4 CALL_FUNCTION            1
              6 RAISE_VARARGS            1
```

A

Reply:
Thanks. While that makes sense, isn't it rather premature optimization?

Reply:
Knuth would be disappointed!

A